### DISCUSSION TITLE
[Feature] Sync specific shows only

### DISCUSSION THREAD

**Discussion by apnar** [arabcoders/watchstate#274] [General] [answered]
I took a look and couldn't find a way to do this so figured I'd make a feature request.  There are a few shows that I watch with someone else that I'd like to sync watched status between our accounts but only those particular shows.  Looks like 'title' is already a database property or maybe do it via one of the parent guids.  Could be a per library property that takes a list of titles or guids as an import/export filter, something like that. 

Looks like I could move them into a separate library on each but that's a lot of overhead and the shows are otherwise not different so that doesn't make much sense.

Thanks for a great tool!

**Comment by arabcoders** [comment, marked as answer]
Hi,

Thank you for your suggestion, unfortunately as you suspected this tool does not support this use case, However you could do it via blacklisting libraries and use one shared library between your targets that contains said tv shows this is the only workable solution at this time. And this tool does not really care about shows titles as they can be different between libraries we work on GUIDs and virtual GUIDs for episodes. i.e. `tvdb://id/(season)/(id)`.

I don't have any good ideas on how to approach this at the moment. we may add global whitelist on import or something. i'll have to think about it
